sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -33,12 +33,12 @@ import org.apache.spark.sql.catalyst.catalog._
 import org.apache.spark.sql.catalyst.catalog.CatalogTypes.TablePartitionSpec
 import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.internal.StaticSQLConf.CATALOG_IMPLEMENTATION
-import org.apache.spark.sql.test.{SharedSQLContext, SQLTestUtils}
+import org.apache.spark.sql.test.{SharedSparkSession, SQLTestUtils}
 import org.apache.spark.sql.types._
 import org.apache.spark.util.Utils
 
 
-class InMemoryCatalogedDDLSuite extends DDLSuite with SharedSQLContext with BeforeAndAfterEach {
```

Comment:
Shall we remove line 25?
```
import org.scalatest.BeforeAndAfterEach
```

Reply:
At the beginning, I removed unused imports in the code files I changed. Later I realized that there are so many files already have unused imports and it's a lot of work to clean unused imports for 170+ files. So I stopped doing it.

Reply:
Ah, I got it. I thought it's related. Never mind then.